vmware-tanzu/nsx-operator
# Add VirtualMachine handler to tag the labels on the related NSX subnet port

This patch will add a watcher in the subnetport controller to watch the VM
event and requeue its own subnetport CR to handle the label change.

Tests done:
1. Create the subnetport CR with VM reference and ensure that the NSX subnet
   port has the tags mapping to the VM labels.
2. Add a label on VM and ensure that the tag is added on the port.
3. Change the existing label on VM and ensure that the port tag is also changed.
4. Remove a label on VM and ensure that the tag on port is removed.
